test/unit/security/test_object_wrapper.py

```diff
@@ -0,0 +1,88 @@
+import pytest
```

Comment:
Can you place this unit test in test/unit/data/security/test_object_wrapper.py instead.

The top-level of test/unit should be directories corresponding to the packages the code belongs in (this is aspirational I know 😅 but it is what I've been working toward). If you place this file in there, it will automatically included in the CI tests for galaxy-data.

Reply:
Of course! Will do.
> The top-level of test/unit should be directories corresponding to the packages the code belongs in 

This makes perfect sense.